chapter_16_javascript.asciidoc

```diff
@@ -46,6 +49,8 @@ NOTE: I'm going to assume you know the basics of JavaScript syntax.
 
 ((("JavaScript testing", "functional test")))
 ((("functional tests (FTs)", "JavaScript", id="FTjava16")))
+
+// RITA: In this first paragraph, please tell us WHY we're adding a new FT. How does this relate to making it so the duplicate item error messages disappear once the user fixes the problem? Give us context.
```

Comment:
the FT literally tests that duplicate item error messages disappear once the user starts fixing the problem 😅 

i know it's not self evident to nonprogrammers!  but maybe the code comments are a guide, even to non-python people:

```
    # Edith starts a list and causes a validation error:
    ...
    # She starts typing in the input box to clear the error
    ...
    # She is pleased to see that the error message disappears
```

Reply:
Got it. I'm coming at this from a purely editorial point of view and making sure the reader is able to follow along with what's happening in the chapter. As a non-programmer, I rely on the narrative text around code examples, so it didn't occur to check the code comments for those context clues.